config/application.yml.default

```diff
@@ -379,7 +377,7 @@ test:
   acuant_max_attempts: '4'
   acuant_passlive_url: https://liveness.example.com
   address_proof_result_lambda_token: test_token_address
-  async_job_refresh_interval_seconds: '1'
+  async_wait_timeout_seconds: '0.25'
```

Comment:
this seems very short?

Reply:
It's limited to the test environment, so I'm open to extending it

Reply:
Oh it's the test environment, that's fine then

Reply:
turns out the redis library only supports integers so I've switched it to 1